tests/skipped_tests_gpu.tbl

```diff
@@ -573,36 +573,21 @@ tests/third_party/cupy/logic_tests/test_comparison.py::TestArrayEqual::test_arra
 tests/third_party/cupy/logic_tests/test_comparison.py::TestArrayEqual::test_array_equal_not_equal
 
 tests/third_party/cupy/manipulation_tests/test_dims.py::TestBroadcast_param_0_{shapes=[(), ()]}::test_broadcast
```

Comment:
please unmute `tests/third_party/cupy/core_tests/test_ndarray_copy_and_view.py::TestArrayCopyAndView::test_astype_strides_broadcast` and update like:
```python
    @testing.for_all_dtypes_combination(("src_dtype", "dst_dtype"))
    @testing.numpy_cupy_equal()
    def test_astype_strides_broadcast(self, xp, src_dtype, dst_dtype):
        src1 = testing.shaped_arange((2, 3, 2), xp, dtype=src_dtype)
        src2 = testing.shaped_arange((2,), xp, dtype=src_dtype)
        src, _ = xp.broadcast_arrays(src1, src2)
        strides = astype_without_warning(src, dst_dtype, order="K").strides
        if xp is numpy:
            strides = tuple(x / src.itemsize for x in strides)
        return strides
```